cmd/kubeadm/app/constants/constants.go

```diff
@@ -27,6 +27,8 @@ const (
 	// KubernetesDir is the directory kubernetes owns for storing various configuration files
 	KubernetesDir = "/etc/kubernetes"
 
+	ManifestsDirName = "manifests"
```

Comment:
nit: I think I'd prefer `ManifestsSubDirName`, sorry for not catching that the first pass